warpcast-langchain/warpcast_api_wrapper.py

```diff
@@ -0,0 +1,37 @@
+import requests
+
+
+class WarpcastApiWrapper:
```

Comment:
Please follow pattern from [TwitterApiWrapper](https://github.com/coinbase/cdp-agentkit/blob/12ddc1175474f4b8a79aeff6d0fb3e51fb4c9aa2/twitter-langchain/twitter_langchain/twitter_api_wrapper.py#L12) where the wrapper is a child class of `BaseModel` and all actions are run using a unified [run_action instance method](https://github.com/coinbase/cdp-agentkit/blob/12ddc1175474f4b8a79aeff6d0fb3e51fb4c9aa2/twitter-langchain/twitter_langchain/twitter_api_wrapper.py#L54)